inlang/source-code/sdk2/src/project/loadProjectFromDirectory.ts

```diff
@@ -28,27 +28,73 @@ export async function loadProjectFromDirectoryInMemory(
 	});
 	await copyFiles({ fs: args.fs, path: args.path, lix: tempProject.lix });
 
-	// TODO loadMessages - call tempProject.lix.settled() to wait for the new settings file
-
-	// TODO loadMessages - since settings are not reactiv yet - just reload the project
+	// TODO call tempProject.lix.settled() to wait for the new settings file, and remove reload of the proejct as soon as reactive settings has landed
+	// NOTE: we need to ensure two things:
+	// 1. settled needs to include the changes from the copyFiles call
+	// 2. the changes created from the copyFiles call need to be realized and lead to a signal on the settings
```

Comment:
I described the issue we are facing with the current setup and left a todo to fix this as soon as settled and reactivty for settings based on changes is implemented. This should not block us from closing https://linear.app/opral/issue/MESDK-163/implement-loadprojectfromdirectory-pluginloadmessages-logic so